### ISSUE TITLE
Replace AlertUseCheck(S1442) with eslint no-alert

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarJS#1078]
Java-based implementation of AlertUseCheck ([RSPEC-1442](https://jira.sonarsource.com/browse/RSPEC-1442)) should be replaced with eslint core rule `no-alert`. It should fix some FPs like these:
```js
function alert() {}
alert();  // FP here
```

([see user complaining about it](https://community.sonarsource.com/t/false-positive-alert-should-not-be-used/2229))

unit test for eslint rule https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-alert.js#L27-L31
